app/models/manageiq/providers/oracle_cloud/cloud_manager/template.rb

```diff
@@ -1,4 +1,12 @@
 class ManageIQ::Providers::OracleCloud::CloudManager::Template < ManageIQ::Providers::CloudManager::Template
+  supports :provisioning do
+    if ext_management_system
+      unsupported_reason_add(:provisioning, ext_management_system.unsupported_reason(:provisioning)) unless ext_management_system.supports_provisioning?
+    else
+      unsupported_reason_add(:provisioning, _('not connected to ems'))
+    end
+  end
```

Comment:
```suggestion
  supports :provisioning do
    if ext_management_system
      ext_management_system.unsupported_reason(:provisioning)
    else
      _('not connected to ems')
    end
  end
```

Reply:
FYI: `unsupported_reason(:feature)` will return `nil` if it is supported
So that is the same as `unsupported_reason(:feature) unless supports?(:feature)`

Reply:
Ha yeah I actually had `ext_management_system.unsupported_reason(:provisioning) unless ext_management_system.supports_provisioning?` thinking you forgot it but then realized it would just return `nil`, slick